app/templates/explore.html

```diff
@@ -109,4 +109,4 @@ <h1 class="header-margin">{{ name|title }}s</h1>
     });
   });
 </script>
-{% endblock %}
+{% endblock %}
```

Comment:
We actually want newlines on files because it means adding new lines to the end won't include an extra removal. Configure your editor to add them.

This goes for all of these files.

Reply:
Ah mb, okok